docs/user/automatic-redirects.rst

```diff
@@ -1,72 +1,41 @@
+and giving you access to define your own redirects.
+
+In this article,
+we explain how our built-in redirects work and what we consider "best practice" for managing incoming links.
 
 .. seealso::
 
    :doc:`/versions`
      Read more about how to handle versioned documentation and URL structures.
 
    :doc:`/user-defined-redirects`
-     If you delete or move a page,
-     you can setup a redirect in place of the old location and choose where users should be redirected.
-
-
-Root URL redirect
------------------
-
-A link to the root of your documentation will redirect to the *default version*,
-as set in your project settings.
-
-This works for both readthedocs.io (|org_brand|), readthedocs-hosted.com (|com_brand|), and :doc:`custom domains </custom-domains>`.
-
-For example::
-
-    docs.readthedocs.io -> docs.readthedocs.io/en/latest/
-    www.pip-installer.org -> www.pip-installer.org/en/latest/
+     Overview of all the redirect features available on Read the Docs.
+     Many of the redirect features are useful either for building external links or handling requests to old URLs.
 
-.. warning::
+   :doc:`/guides/redirects`
+     How to add a user-defined redirect, step-by-step.
+     Useful if your content changes location!
 
-   This only works for the root URL, not for internal pages.
-   It's designed to redirect people from ``http://pip.readthedocs.io/`` to the default version of your documentation,
-   since serving up a 404 here would be a pretty terrible user experience.
 
-.. note::
-   If the "develop" branch was designated as the default version,
-   then ``http://pip.readthedocs.io/`` would redirect to ``http://pip.readthedocs.io/en/develop``.
-   But, it's not a universal redirecting solution.
-   So, for example, a link to an internal page like
-   ``http://pip.readthedocs.io/usage.html`` doesn't redirect to ``http://pip.readthedocs.io/en/latest/usage.html``.
+Best practice: "permalink" your pages
+-------------------------------------
 
-   The reasoning behind this is that RTD organizes the URLs for docs so that multiple translations and multiple versions of your docs can be organized logically and consistently for all projects that RTD hosts.
-   For the way that RTD views docs,
-   ``http://pip.readthedocs.io/en/latest/`` is the root directory for your default documentation in English, not ``http://pip.readthedocs.io/``.
-   Just like ``http://pip.readthedocs.io/en/develop/`` is the root for your development documentation in English.
-
-Among all the multiple versions of docs,
-you can choose which is the "default" version for RTD to display,
-which usually corresponds to the git branch of the most recent official release from your project.
-
-Are "permalinks" possible?
---------------------------
-
-You might be familiar with permalinks from blogging.
+You might be familiar with :term:`permalinks <permalink>` from blogging.
```

Comment:
it's a very good article that I used as inspiration for the glossary entry :)

but yeah, I'll trim that entry since it gives the impression that it's a term that RTD has a special definition of.